src/main/java/org/isf/opd/manager/OpdBrowserManager.java

```diff
@@ -89,10 +89,6 @@ protected void validateOpd(Opd opd, boolean insert) throws OHDataValidationExcep
 		// Check Ward
 		if (ward == null) {
 			errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.common.pleaseselectaward.msg")));
-		} else {
-			if (!ward.isOpd()) {
-				errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.opd.specifiedwardisnotenabledforopdservice.msg")));
-			}
 		}
```

Comment:
Why to remove this check? this is important in case the API are called directly

Reply:
ah okay yes

Reply:
we can close this PRs